lib/bootloader_setup.pm

```diff
@@ -319,6 +324,61 @@ sub specific_bootmenu_params {
     }
 }
 
+sub bootmenu_edit_uefi_grub {
+    send_key "e";
+    # assume bios+grub+anim already waited in start.sh
+    # in grub2 it's tricky to set the screen resolution
+    if (is_jeos) {
+        for (1 .. 3) { send_key "down"; }
+        send_key "end";
+        # delete "800x600"
+        for (1 .. 7) { send_key "backspace"; }
+    }
+    else {
+        for (1 .. 2) { send_key "down"; }
+        send_key "end";
+        # delete "keep" word
+        for (1 .. 4) { send_key "backspace"; }
+    }
+    # hardcoded the value of gfxpayload to 1024x768
+    type_string "1024x768";
+    assert_screen "gfxpayload_changed", 10;
+    # back to the entry position
+    send_key "home";
+    for (1 .. 2) { send_key "up"; }
+    if (is_jeos) {
+        send_key "up";
+    }
+    sleep 5;
+    for (1 .. 4) { send_key "down"; }
+    send_key "end";
+
+    if (get_var("NETBOOT")) {
+        type_string_slow " install=" . get_netboot_mirror;
+        save_screenshot();
+    }
+    send_key "spc";
+
+    # if(get_var("PROMO")) {
```

Comment:
I did not created it the commented code, I just drag it from on place to another.